config/api-service.php

```diff
@@ -12,4 +12,9 @@
             'enable_policy' => true,
         ],
     ],
+    'tenancy' => [
+        'enabled'   => false,
+        'is_tenant_aware' => false,
```

Comment:
it’s not duplicate, when using tenancy.enabled = true 
This means this package will return only resources which tenant user has access to. If false then no tenancy filtering at all. 

If you also enable tenancy_aware to true the routes change and you can only fetch a specific tenant id resource. So then you only retrieve data of one tenant resource based on the tenant ID/slug in de api route url. 
hope that makes sense.

Reply:
What could be added is that when a resource has:
protected static bool $isScopedToTenant = false;
then even if config “is_tenant_aware” = true than for that resource you should return all records.

Reply:
okay, but it will be error (there is no api generated) if you set the `enabled=false` & `is_tenant_aware=true`, maybe u can throw some exception for this case. So, developers know what happend.